develop/dev-best-practices.md

```diff
@@ -81,7 +81,7 @@ keep image size small:
 
 ## Use CI/CD for testing and deployment
 
-- When you check a change into source control or create a pull request, use
+- When you check-in a change to source control or create a pull request, use
```

Comment:
Could you remove the hyphen in check-in? Use `check in` for verbs (without the hyphen).

Reply:
done